README.md

```diff
@@ -1 +1,8 @@
-# nmstate
\ No newline at end of file
+# nmstate
+
+Run all checks:
+
+```shell
+pip install tox pbr
+tox
+```
```

Comment:
Please also show how to use the executable.

Reply:
Done